cds_migrator_kit/rdm/migration/cli.py

```diff
@@ -67,8 +67,12 @@ def stats():
     "--filepath",
     help="Path to the list of records file that the legacy statistics will be migrated.",
 )
+@click.option(
+    "--less-than-date",
+    help="ISO string e.g 2024-12-13T20:00:00 to migrate events up to this date.",
```

Comment:
just for my knowledge update, which date will we use finally? the time we applied the redirection or the migration time?

Reply:
TBD but I think it should be the redirection time